.github/ISSUE_TEMPLATE/icon_request.md

```diff
@@ -4,17 +4,23 @@ about: Request a new icon for SimpleIcons
 labels: new icon
 ---
 
-<!-- Before opening a new issue search for duplicate or closed issues -->
+<!--
+  We won't add non-brand icons or anything related to illegal services.
+  If in doubt, open an issue and we'll have a look.
 
+  Before opening a new issue please search for duplicate or closed
+  issues or PRs. If you find one for the brand your requesting then
+  leave a comment on it or add a reaction to the original post.
 
-<!-- When requesting a new icon we need information such as: -->
+  When requesting a new icon please provide the following information:
+-->
 **Name:**
 **Website:**
+**Alexa rank:**
+  <!-- The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/
+       Please see our contributing guidelines at
+       https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
```

Comment:
I'm still not sure if I like having the link here... Would you be open to just omitting it and hope contributors can find it themselves? Or maybe put the link at the bottom/top of the template and refer to the links there?

We could also say there's a link to the contributing guidelines in the sidebar, buy I'm personally not a big fan of that either is it depends on a design choice we have no control over.

Any opinions on this from other @simple-icons/maintainers? If most are fine with the current approach I'm fine with leaving it like this as well :)

Reply:
I'd say either omit it, or add it at the bottom.

Reply:
Done :+1: Let me know what ye think of the wording